graphistry/Plottable.py

```diff
@@ -22,17 +26,28 @@
         from sklearn.pipeline import Pipeline
     except:
         Pipeline = Any
+    try:
+        from cugraph import Graph
+        from cugraph import MultiGraph
+        from cugraph import BiPartiteGraph
```

Comment:
tricky question here of avoiding heavy imports unless we want them, so keeping as `'..'` in `Plottable.py` seems better?

Reply:
these are under `if TYPE_CHECKING`